faker.go

```diff
@@ -170,7 +170,9 @@ func (f Faker) Lorem() Lorem {
 }
 
 func (f Faker) Person() Person {
-	return Person{&f}
```

Comment:
Don't need to format, if you think that this is important please open a new PR formatting all source code at once